server.js

```diff
@@ -23,6 +24,7 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 
 
+
```

Comment:
Next Step: You will need to require your models here for your routes. `var db = require('./models');`